apps/docs-website/docs/eslint-plugins.md

```diff
@@ -4,29 +4,29 @@ sidebar_position: 7
-- [eslint-plugin-storybook](https://github.com/storybookjs/eslint-plugin-storybook)
-- [eslint-plugin-astro](https://github.com/ota-meshi/eslint-plugin-astro)
-- [eslint-plugin-fsecond](https://github.com/AndreaPontrandolfo/eslint-plugin-fsecond)
-- [@regru/eslint-plugin-prefer-early-return](https://github.com/regru/eslint-plugin-prefer-early-return)
-- [eslint-plugin-arrow-return-style](https://github.com/u3u/eslint-plugin-arrow-return-style)
-- [eslint-plugin-simple-import-sort](https://github.com/lydell/eslint-plugin-simple-import-sort)
+- [`@eslint/js`](https://www.npmjs.com/package/@eslint/js)
+- [`typescript-eslint`](https://typescript-eslint.io)
+- [`@stylistic/eslint-plugin`](https://eslint.style)
+- [`eslint-plugin-react`](https://github.com/jsx-eslint/eslint-plugin-react)
+- [`eslint-plugin-jsx-a11y`](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y)
+- [`eslint-plugin-react-hooks`](https://github.com/facebook/react/tree/main/packages/eslint-plugin-react-hooks)
+- [`eslint-plugin-react-refresh`](https://github.com/ArnaudBarre/eslint-plugin-react-refresh)
+- [`@eslint-react/eslint-plugin`](https://github.com/rel1cx/eslint-react)
+- [`eslint-plugin-unicorn`](https://github.com/sindresorhus/eslint-plugin-unicorn)
+- [`eslint-plugin-regexp`](https://github.com/ota-meshi/eslint-plugin-regexp)
+- [`eslint-plugin-sonarjs`](https://github.com/SonarSource/SonarJS/tree/master/packages/jsts/src/rules#readme)
+- [`eslint-plugin-jsdoc`](https://github.com/gajus/eslint-plugin-jsdoc)
+- [`eslint-plugin-tsdoc`](https://tsdoc.org/pages/packages/eslint-plugin-tsdoc/)
+- [`eslint-plugin-jest`](https://github.com/jest-community/eslint-plugin-jest)
+- [`@vitest/eslint-plugin`](https://github.com/vitest-dev/eslint-plugin-vitest)
+- [`eslint-plugin-import`](https://github.com/import-js/eslint-plugin-import) with [`eslint-import-resolver-typescript`](https://github.com/import-js/eslint-import-resolver-typescript)
+- [`eslint-plugin-lodash-f`](https://github.com/AndreaPontrandolfo/eslint-plugin-lodash) (my fork of [`eslint-plugin-lodash`](https://github.com/wix/eslint-plugin-lodash))
+- [`eslint-plugin-remeda`](https://github.com/AndreaPontrandolfo/eslint-plugin-remeda)
+- [`@next/eslint-plugin-next`](https://nextjs.org/docs/app/api-reference/config/eslint)
+- [`eslint-plugin-playwright`](https://github.com/playwright-community/eslint-plugin-playwright)
+- [`eslint-plugin-storybook`](https://github.com/storybookjs/eslint-plugin-storybook)
+- [`eslint-plugin-astro`](https://github.com/ota-meshi/eslint-plugin-astro)
+- [`eslint-plugin-fsecond`](https://github.com/AndreaPontrandolfo/eslint-plugin-fsecond)
+- [`@regru/eslint-plugin-prefer-early-return`](https://github.com/regru/eslint-plugin-prefer-early-return)
+- [`eslint-plugin-arrow-return-style`](https://github.com/u3u/eslint-plugin-arrow-return-style)
+- [`eslint-plugin-simple-import-sort`](https://github.com/lydell/eslint-plugin-simple-import-sort)
```

Comment:
Fair enough. Is wrapping them in backticks acceptable?

Reply:
eh, i would rather not. It would be fine either way, but i don't think it's a needed change. In this case they represent more "the technical name of a library" rather than a piece of code, plus they already have the styling of links.